src/redux/reducers/authReducer.test.ts

```diff
@@ -0,0 +1,45 @@
+import MockFactory from "../../common/mockFactory";
+import { reducer } from "./authReducer";
+import { IAuth } from "../../models/applicationState";
+import {signInAction, signOutAction} from "../actions/authActions";
+import { anyOtherAction } from "../actions/actionCreators";
+
+describe("Auth Reducer", () => {
+    it("Saves auth with new access token", () => {
+        const testAuth = MockFactory.createTestAuth();
+        const state: IAuth = testAuth;
+        const newAuth = MockFactory.createTestAuth("new_auth");
+        const action = signInAction(newAuth.accessToken);
+
+        const result = reducer(state, action);
+        expect(result).not.toBe(state);
```

Comment:
what about checking that the `access_token` is set to the `newAuth.accessToken` ?

like below...